CalculatorTests/UnitTest1.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using Xunit;
+using CalculatorWithTests;
+
+namespace CalculatorTests
+{
+    public class UnitTest1
+    {
+        [Theory]
+        [InlineData (10, 20, 30)]
+        [InlineData(20, 30, 50)]
+        [InlineData(30, 40, 70)]
+        public void AddNumbers(int number1, int number2, int expected)
+        {
+            // Arrange
+            Calculator calculator = new Calculator();
+
+            // Act
+            int actual = calculator.AddNumbers(number1, number2);
+
+
+            // Assert
+            Assert.Equal(expected, actual);
+        }
+        [Theory]
+        [InlineData (5, 3, 2)]
+        [InlineData(10, 8, 2)]
+        [InlineData(20, 18, 2)]
+        public void Subtract(int number1, int number2, int expected)
+        {
+            // Arrange
+            Calculator calculator = new Calculator();
+
+            // Act
+            int actual = calculator.Subtract(number1, number2);
+
+
+            // Assert
+            Assert.Equal(expected, actual);
+        }
+        [Theory]
+        [InlineData (2, 4, 8)]
+        [InlineData(4, 6, 24)]
+        [InlineData(6, 8, 48)]
+        public void MultiplyNumbers (int number1, int number2, int expected)
+        {
+            // Arrange
+            Calculator calculator = new Calculator();
+
+            // Act
+            int actual = calculator.Multiply(number1, number2);
+
+
+            // Assert
+            Assert.Equal(expected, actual);
+        }
+        [Theory]
+        [InlineData (100, 50, 2)]
+        [InlineData (50, 25, 2)]
+        [InlineData (20, 1, 20)]
+        public void DivideNumbers (int number1, int number2, int expected)
```

Comment:
There's a suspiciously absent test cast, which is when dividing by zero. The tests should establish what the method being tested is supposed to return if given various inputs, including edge cases such as zero for a divisor